delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/Storages.scala

```diff
@@ -368,10 +335,6 @@ object Storages {
       s.copy(rev = e.rev, value = e.value, source = e.source, updatedAt = e.instant, updatedBy = e.subject)
     }
 
-    def tagAdded(e: StorageTagAdded): Option[StorageState] = state.map { s =>
```

Comment:
I think we still need to compute a new state and update the rev, we may break some history otherwise.
For example, this should still work:
* StorageCreated
* StorageTagged
* StorageUpdated
We should still have a storage with revision 3